test/unit/modules/validation/validation-module-manager.test.js

```diff
@@ -0,0 +1,82 @@
+import { describe, it, beforeEach } from 'mocha';
+import { expect, assert } from 'chai';
+import { readFile } from 'fs/promises';
+import { calculateRoot } from 'assertion-tools';
+import ValidationModuleManager from '../../../../src/modules/validation/validation-module-manager.js';
+import Logger from '../../../../src/logger/logger.js';
+
+let validationManager;
+
+const config = JSON.parse(await readFile('./test/unit/modules/validation/config.json', 'utf-8'));
+const assertion = [
+    {
+        '@context': 'https://schema.org',
+        '@id': 'https://tesla.modelX/2321',
+        '@type': 'Car',
+        name: 'Tesla Model X',
+        brand: {
+            '@type': 'Brand',
+            name: 'Tesla',
+        },
+        model: 'Model X',
+        manufacturer: {
+            '@type': 'Organization',
+            name: 'Tesla, Inc.',
+        },
+        fuelType: 'Electric',
+    },
+];
+const hashFunctionId = 1;
+
+describe('Validation module manager', async () => {
+    beforeEach('initialize validation module manage', async () => {
+        validationManager = new ValidationModuleManager({
+            config,
+            logger: new Logger(),
+        });
+
+        expect(await validationManager.initialize()).to.be.true;
+    });
+
+    it('validates module name', async () => {
+        const moduleName = await validationManager.getName();
+
+        expect(moduleName).to.equal('validation');
+    });
+
+    it('validate calculation of root hash are matched', async () => {
+        const expectedRootHash = calculateRoot(assertion);
```

Comment:
Can you add unit tests that will cover error handling if we send null as assertion, also add those checks for all methods